build/rollup.config.js

```diff
@@ -33,7 +33,7 @@ if (BUNDLE) {
 module.exports = {
   input: path.resolve(__dirname, '../js/src/index.js'),
   output: {
-    banner,
+    banner: banner(),
```

Comment:
```suggestion
    banner: banner(),
    interop: 'amd',
```

Reply:
See https://github.com/rollup/plugins/pull/1358

Reply:
I'm not seeing this as an option in
https://rollupjs.org/guide/en/#outputinterop
and it does seem to break the build locally.